Tribler/Core/Modules/restapi/debug_endpoint.py

```diff
@@ -12,6 +14,12 @@
 import Tribler.Core.Utilities.json_util as json
 
 
+class MemoryDumpBuffer(StringIO):
```

Comment:
Why do we need this `MemoryDumpBuffer` exactly?

Reply:
`meliae` expects its file handle to support `write()`, `flush()` and `__call__()`, the `StringIO` class does not support the latter.

Reply:
Could you add a comment in the code for that? 👍

Reply:
Done 👍